specs/movie_reserver_spec.rb

```diff
@@ -75,27 +75,77 @@
     end
   end
 
-  xdescribe 'rent_movie' do
+  describe 'rent_movie' do
     it 'returns a rental for a successfully rented movie' do
-      # TODO Your Code goes here
+      #Setup
+      movie_reserver  = GreenBox::MovieReserver.new
+      title = 'Crazy Rich Asians'
+      date_range = GreenBox::DateRange.new(Time.parse('2018-08-08'),Time.parse('2018-08-09'))
+      customer_name = ' Sabine '
 
+      #Method under testn(call the things that I am verifying)
+      rental = movie_reserver.rent_movie(title, date_range, customer_name)
+
+      #Assertions - something that has changed that tells
+      #us we have the expected result
+      expect(rental).wont_be_nil
```

Comment:
This is good, you also might want to test that
`expect(rental.movie.title).must_equal title` (the title of the movie being rented is right.

You can also test that the number of rentals has increased by one (so the length of `rentals` increased by one).